code/script.js

```diff
@@ -1,47 +1,317 @@
-// All the DOM selectors stored as short variables
+//Global variables
+
 const chat = document.getElementById('chat')
+const sendBtn = document.querySelector(".send-btn");
+const inputWrapper = document.querySelector(".input-wrapper");
+let defaultForm = document.getElementById("name-form");
+
+//Global variables - to store address and time bookig from user
+let address = "";
+let time = "";
+
+
+//Music
+const pop = new Audio("pop.flac");
 
-// Global variables, if you need any, declared here
+//Video control
+const videoBtn = document.querySelector(".video-btn");
+const video = document.querySelector(".video")
+
+videoBtn.addEventListener("click", function(){
+    videoBtn.classList.toggle("slide")
+    if (videoBtn.classList.contains("slide")){
+      video.pause()} 
+      else {
+        video.play() } 
+    })
 
-// Functions declared here
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
-  if (sender === 'user') {
-    chat.innerHTML += `
-      <section class="user-msg">
-        <div class="bubble user-bubble">
-          <p>${message}</p>
-        </div>
-        <img src="assets/user.png" alt="User" />  
-      </section>
-    `
-  } else if (sender === 'bot') {
-    chat.innerHTML += `
-      <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
-        <div class="bubble bot-bubble">
-          <p>${message}</p>
-        </div>
-      </section>
-    `
+    if (sender === 'user') {
+      chat.innerHTML += `
+        <section class="user-msg">
+          <div class="bubble user-bubble">
+            <p>${message}</p>
+          </div>
+          <img src="https://images.pexels.com/photos/4113971/pexels-photo-4113971.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260" alt="User" />  
+        </section>
+      `
+    } else if (sender === 'bot') {
+      chat.innerHTML += `
+        <section class="bot-msg">
+          <img src="https://images.unsplash.com/photo-1634909924531-4daae117dbc1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2062&q=80" alt="Bot" />
+          <div class="bubble bot-bubble">
+           <p class="bot-text">${message}</p>
+          </div>
+        </section>
+      `
+    }
+    // This little thing makes the chat scroll to th/e last 
```

Comment:
Maybe this comment could be "cleaned" up a little bit, there's several forward slashes inside the message?